src/openstudio_app/CMakeLists.txt

```diff
@@ -177,6 +177,11 @@ if( APPLE )
     VERBATIM
   )
 
+  #add_custom_command(TARGET ${target_name}
+  #  POST_BUILD
+  #  COMMAND ${QT_MACDEPLOY_QT} $<TARGET_BUNDLE_DIR:${target_name}> -verbose=2
+  #)
```

Comment:
Should we use macdeployqt, and linuxdeployqt?

Reply:
Eventually I think so.  When I ran macdeployqt the resulting package crashed, I don't know why.